vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/VGSCheckoutOnInitListener.kt

```diff
@@ -0,0 +1,23 @@
+package com.verygoodsecurity.vgscheckout
+
+import com.verygoodsecurity.vgscheckout.exception.VGSCheckoutException
+
+/**
+ * Interface definition for callbacks that are invoked when VGS Checkout initialization succeeds or fails.
+ */
+interface VGSCheckoutOnInitListener {
+
+    /**
+     * Called when initialization of the player succeeds.
+     *
+     * @return True if the VGS Checkout should start the payment form, false otherwise.
+     */
+    fun onCheckoutInitializationSuccess(): Boolean
```

Comment:
@Endorf I'm a little bit confused with this return type, can you share some examples when users returns false?

Reply:
- when a user closes checkout till the actions are completed
- the initialization was wrong.
- Any other reasons related to application flow.

Adding config as an attribute could make sens

Reply:
@Endorf Good points, but in this case, I suggest using VGSCheckoutCancellable for this purpose.

Reply:
yeah. It makes sens with onCheckoutInitializationFailure.